data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +518,1020 @@ mission "Unfettered Jump Drive Trading"
+			`	Then you are taken far away by a beam of pure light. You see collapsed civilisations, fighting to reclaim their forgotten glory as they struggle to feed their own people. You see a planet, once beautiful, now being bombarded until not a single trace of life is left. You see stars, seemingly eternal, blasted away with all surrounding space. You see automated ships, repairing wounds.`
+			action
+				set "unfettered: heshi trip"
+				"unfettered rep: Si-Ra" ++
+			`	Except you never wake up. The final words you hear before all is truly dark speak of an ongoing medical crisis taking priority.`
+				to display
+					not "Unfettered Small Medicals Rush: done"
+					not "Unfettered Big Medicals Rush: done"
+					not "<uhai> 1: Bring Medicine: done"
+				die
+			`	Until, finally you are brought back by that same beam of light, and it is all you can see.`
+			choice
+				`	(Wake up.)`
+				`	(Give up.)`
+					die
+			`	You wake up in a hospital bed, the Captain is there and speaks very quietly, almost gently, "That was maybe not smart, lad, but it sure was brave! You have what it takes to be one of us, just make sure you don't get killed next time." He then moves to the door behind you.`
+				goto end
+			label run
+			action
+				log "Defeated an Unfettered Captain in a duel, but it seems that was not the whole test. The second part involved a monstrous scorpion from which it felt like a good idea to run at the time. The Unfettered did not see it that way, however."
+			`	You start running away and hear the Captain speak to his crew, loud enough for you to hear, "The monkey is running away from danger. It would not be fit to join us indeed. It will not fight for us."`
+				decline
+			label shoot
+			`	You get your weapon out and shoot the monster as it attempts to rush out of its aquarium. You are not sure if your shot over the aquarium hit it, because of the erupting water that came out your way as the beast emerged, but at least you are still alive!`
+				goto reward
+
+event "unfettered: JD raid plan ready"
+
+
+
+event "cerberus revealed"
+	government "Cerberus"
+		remove "display name"
+
+# SPOILER: stealing JD from a vault guarded by the Alpha's indoctrinated pirates: Cerberus
+mission "<uhai> 3: Recover Jump Drives"
+	name `Recover the jump drives`
+	description `Go to the pirate world of <stopovers> and recover 4 jump drives from a vault.`
+	source "Darkcloak"
+	stopover "Freedom"
+	clearance
+	passengers 1
+	to offer
+		has "event: unfettered: JD raid plan ready"
+	on offer
+		conversation
+			`As you pass what looks like a bar in the spaceport, you're greeted by the same Hai captain that challenged you earlier. "So we meet again! Arr, if ye be wanting to scuttle me clipper again, it'll take a lot more salt out of ye! Well... once I have her fixed and back, anyway. But that not be why I be here, today," he says, apparently happy to see you. "I have a job for ye!"`
```

Comment:
```suggestion
			`As you pass what looks like a bar in the spaceport, you hear a familiar voice shout out. "There ye be!" It's the same Hai captain that challenged you earlier. "So we meet again! Arr, if ye be wanting to scuttle me clipper again, it'll take a lot more salt out of ye! Well... once I have her fixed and back, anyway. But that not be why I be here, today," he says, apparently happy to see you. "I have a job for ye!"`
```

Reply:
monkey is a way to insult you indirectly by not recognizing you, so maybe I should make him say that if he doesnt like you

Reply:
It felt like a good segue as shortly after you call him "my favorite squirrel" and then this, `He does not seem to react to the insult. He did call you monkey a few times before, so it's only fair for you to return the courtesy` 
Also, at this point I don't think they are friends yet as they are not on a first name basis.